pkg/operator/starter.go

```diff
@@ -152,6 +153,9 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		statusControllerOptions = append(statusControllerOptions, apiservercontrollerset.WithStatusControllerPdbCompatibleHighInertia("APIServer"))
 	}
 
+	infraLister := configInformers.Config().V1().Infrastructures().Lister()
+	infraInformer := configInformers.Config().V1().Infrastructures().Informer()
```

Comment:
it could be removed if we do https://github.com/openshift/cluster-openshift-apiserver-operator/pull/489/files#r783729064